azure-go-appservice-docker/README.md

```diff
@@ -48,9 +48,33 @@ The example shows two scenarios:
 
     ```
     $ pulumi stack output helloEndpoint
-    http://hello-app-91dfea.azurewebsites.net/hello
```

Comment:
I changed the image to an image that exists so I updated the README with the expected outputs.